shipyard_airflow/plugins/openstack_operators.py

```diff
@@ -42,18 +41,15 @@ def __init__(self,
         self.xcom_push_flag = xcom_push
 
     def execute(self, context):
-        logging.info("Running OpenStack Command: " + self.openstack_command)
+        logging.info("Running OpenStack Command: " + ' '.join(self.openstack_command))
 
-        # Emulate "source" in bash. Sets up environment variables.
+        # Build environment variables.
         pipe = subprocess.Popen(". %s; env" % self.openrc_file, stdout=subprocess.PIPE, shell=True)
```

Comment:
The current assumption is that the rc file will contain only the environment information that is needed to access the OpenStack services.  Hence we will just need read that information.